R/subset_datarqst.R

```diff
@@ -30,73 +42,189 @@
 #' subset_datarqst(Species = c("PARMAJ", "FICALB"))
 #'
 #' #Create a specific subset of species and population combinations
-#' subset_datarqst(filter = c("HOG_PARMAJ", "VEL_FICALB", "VLI_PARMAJ"))
+#' subset_datarqst(PopSpec_Filter = c("HOG_PARMAJ", "VEL_FICALB", "VLI_PARMAJ"))
 #'
 #' }
 
 subset_datarqst <- function(file = file.choose(),
                             PopID = unique(pop_names$code),
                             Species = unique(Species_codes$Code),
-                            filter = NULL){
+                            PopSpec_Filter = NULL,
+                            include_conflicting = FALSE,
+                            output_type = "R",
+                            save = TRUE,
+                            test = FALSE){
+
+  if(!test){
+    message(crayon::bold(crayon::green("Select standard data file (RDS format).")))
+    standard_data <- readRDS(file = file)
+  }else{
+    message(crayon::green("Using 'pipeline_output' as standard data."))
+    standard_data <- pipeline_output
+  }
```

Comment:
This should be a fine workaround for now, but might cause some issues in `devtools::check()`. I will deal with this later though.